src/components/card/card.js

```diff
@@ -1,7 +1,52 @@
+import { ACTION } from "../../action/action";
+import { useGlobal } from "../../context/globalContext";
+import { findItem } from "../../utils/findItem";
+import { useState } from "react";
 import "./card.css";
 
 export const Card = ({ video }) => {
   const { title, anime, _id } = video;
+  const {
+    globalState: { likeVideo, watchLater },
+    globalDispatch,
+  } = useGlobal();
```

Comment:
- I'm not sure what 'UseGlobal' context is for, prefer using 'UseVideo' or something.

Reply:
fixed it